src/editors/multiselect.js

```diff
@@ -8,12 +8,17 @@ JSONEditor.defaults.editors.multiselect = JSONEditor.AbstractEditor.extend({
     var items_schema = this.jsoneditor.expandRefs(this.schema.items || {});
 
     var e = items_schema["enum"] || [];
+	var eTitles = items_schema.options.enum_titles || items_schema["enum"] || [];
+	
+	
     this.option_keys = [];
+	this.option_titles = [];
     for(i=0; i<e.length; i++) {
       // If the sanitized value is different from the enum value, don't include it
       if(this.sanitize(e[i]) !== e[i]) continue;
 
       this.option_keys.push(e[i]+"");
+	  this.option_titles.push(eTitles[i]+"");
```

Comment:
I'm not actually a maintainer so I'm not sure what else to do to advance this. I was just trying to be helpful since i'm hoping this gets merged.

As far as conflicts, I had noticed that github is indicating that it can't auto-merge these changes. It might simply be because your changes are off an older version of master. You can update your fork to the latest master https://help.github.com/articles/syncing-a-fork/